### ISSUE TITLE
Uncaught TypeError: this.scrollTo is not a function

### ISSUE THREAD

**Issue by WillieMaddox** [GaborFarkas/mastering_openlayers3#11]
I just noticed that I get this error when editing a layer name.

``` javascript
layerSpan.addEventListener('blur', function () {
    if (this.contentEditable) {
        this.contentEditable = false;
        layerDiv.draggable = true;
        layer.set('name', this.textContent);
        layerDiv.classList.add('ol-unselectable');
        layerDiv.title = this.textContent;
        this.scrollTo(0, 0);
    }
});
```

The error occurs on the `this.scrollTo(0, 0)` line.

I tried googling but everything I'm seeing is related to jquery.  Do you have any suggestions for a fix?

**Comment by WillieMaddox** [comment]
I think 

``` javascript
this.scrollLeft = 0;
```

will work.

**Comment by GaborFarkas** [comment]
Good catch, I assume you ran into this error using Firefox. It seems like only the window element has a standardised [scrollTo](https://developer.mozilla.org/en-US/docs/Web/API/Window/scrollTo) method, other elements only have it in WebKit powered browsers, such as Opera or Chrome.

**Comment by WillieMaddox** [comment]
Actually, I'm seeing this in chrome.  Haven't checked with other browsers.

**Comment by lukejpreston** [comment]
I know this is old and possibly unrelated but came across when google searching

`scrollTo` doesn't seem to exist on chrome version 57 on `documents`, it does exist on `window` but using `scrollLeft` seems to fix the issue for me. As an aside `scrollTop` also does not exist on `documents` but does on `window` in chrome version 57

**Comment by rudyxu1102** [comment]
`scrollTo` also doesn't work in chrome version 59 on `document`.  But `scrollTop` can use in chrome version 59.

**Comment by GaborFarkas** [comment]
Thank you for the details. Please feel free to create a PR if you would like to see this fixed ASAP.

**Comment by yossi-shasho** [comment]
scroll and scrollTo don't work on various android browsers. scrollLeft = 0 fixed this issue for me
